src/overlays/actors/ovl_En_Rd/z_en_rd.c

```diff
@@ -794,8 +794,8 @@ void EnRd_Update(Actor* thisx, GlobalContext* globalCtx) {
 
     func_80AE4114(this, globalCtx);
 
-    if (gSaveContext.unk_1422 != 0 && this->unk_318 == 0) {
-        gSaveContext.unk_1422 = 0;
+    if (gSaveContext.sunsState != SUNS_INACTIVE && this->unk_318 == 0) {
```

Comment:
is this sun's song? I would have it in full `sunSongState` / `SUNSONG_INACTIVE`

Reply:
when i named this i was purposefully trying to avoid possible confusion about it being about the state of the actual song, if that makes any sense. Like this is only for the thing that happens after the song is done playing

I agree it would probably be better though and should be fine to understand, just explaining the thought process a bit